src/main/java/calculator/CalculatorApplication.java

```diff
@@ -0,0 +1,20 @@
+package calculator;
+
+
+import java.util.Scanner;
+
+import static calculator.service.CalculatorProcessService.processCalculator;
+import static calculator.view.Input.getInput;
+import static calculator.view.output.printResult;
+
+public class CalculatorApplication {
+
+    private static final Scanner STRING_INPUT = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        System.out.println("문자열 입력 : ");
+        String[] inputs = getInput(STRING_INPUT.nextLine());
```

Comment:
```suggestion
    public static void main(String[] args) {
        List<String> inputs = getInput();
```

입력관련 클래스 `Input`을 만드셨는데 Input에게 맡겨보는 게 어떨까요?

Reply:
Input으로 처리할려고하면 헤당 
public static String[] getInput(String inputString) throws IllegalArgumentException {
        validate(inputString);
        return inputString.split(" ");
    }

    private static void validate(String inputString) {
        if(inputString == null || inputString.isBlank()){
            throw new IllegalArgumentException("입력 값이 Null이거나 빈 공백 문자입니다.");
        }
    }Input클래스에서 사용자가 입력한 문자열을 처리할수없어서 Main쪽에 넣었는데 어떻게 처리해야할까요?